.github/workflows/release.yml

```diff
@@ -3,23 +3,56 @@ name: Release
 on:
     workflow_dispatch:
       inputs:
-        tag_name:
-          description: 'Proxy lib tag name + version'
+        proxy_lib:
+          description: 'Proxy lib name'
           required: true
+        proxy_lib_version:
+          description: 'Proxy lib version'
+          required: true  
+        min_version_supported:
+          description: 'Original package last supported version'
+          required: true    
 
 jobs:
+    test:
+        uses: ./.github/workflows/test.yml
+        secrets:
+          ACTIONS_GITHUB_TOKEN: ${{ secrets.ACTIONS_GITHUB_TOKEN }}
     tag:
-        needs: Test
         runs-on: ubuntu-latest
+        needs: [test]
         env:
           GH_ACCESS_TOKEN: ${{ secrets.ACTIONS_GITHUB_TOKEN }}
 
         steps:
           - name: Checkout Repo
             uses: actions/checkout@v3
 
-          - name: tag and push
-            uses: actions-ecosystem/action-push-tag@v1
+          - name: Tag and push
+            run: |
+              git tag proxy-libs/${{ github.event.inputs.proxy_lib }}/${{ github.event.inputs.proxy_lib_version }} && git push origin proxy-libs/${{ github.event.inputs.proxy_lib }}/${{ github.event.inputs.proxy_lib_version }}
+          
+          - name: Configure AWS credentials
+            uses: aws-actions/configure-aws-credentials@13d241b293754004c80624b5567555c4a39ffbe3
             with:
-              tag: ${{ github.event.inputs.tag_name }}
-        
\ No newline at end of file
+              aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+              aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+              aws-region: us-east-1
+
+          - name: Get current proxy_libs_version.json
```

Comment:
Why not use a bucket that reflected by a cdn? Thats would improve the performance of pulling the json

Reply:
I just used a public bicket.. we can do that